RSGIS_M.py

```diff
@@ -54,8 +54,7 @@
 import numpy as np
 import os
 from osgeo import ogr, osr
-import gdal
```

Comment:
`import gdal` also need to be changed to `from osgeo import gdal`

Reply:
Thanks for reviewing! I deleted that line in the pr, as line 53 already had that import statement.